src/main/java/com/github/cameltooling/lsp/internal/completion/modeline/CamelKModelineOptionNames.java

```diff
@@ -38,8 +38,17 @@ private CamelKModelineOptionNames() {}
 	public static final String OPTION_NAME_RESOURCE = "resource";
 	public static final String OPTION_NAME_OPEN_API = "open-api";
 	
+	private static final String DESCRIPTION_SINCE_COMMUNITY_1_5_PROD_1_4 = "Option is available since Community 1.5 and Red Hat productized 1.4";
+	
 	static {
 		OPTION_NAMES_WITH_DESCRIPTION = new HashMap<>();
+		OPTION_NAMES_WITH_DESCRIPTION.put("build-property",
+				"Add a build time property or properties file (syntax: [my-key=my-value|file:/path/to/my-conf.properties]\n\n"
+				+ DESCRIPTION_SINCE_COMMUNITY_1_5_PROD_1_4);
+		OPTION_NAMES_WITH_DESCRIPTION.put("config",
+				"Add a runtime configuration from a Configmap, a Secret or a file (syntax: [configmap|secret|file]:name[/key],"
```

Comment:
Might change the text slightly to remove the unnecessary indefinite articles in the list... "Add a runtime configuration from a ConfigMap, Secret, or file..."

Reply:
this text is duplicated from upstream https://camel.apache.org/camel-k/latest/cli/modeline.html I will provide the suggestion and if accepted, update here to have it in phase

Reply:
https://github.com/apache/camel-k/pull/2451

Reply:
merged in master upstream.
updated PR